packages/api/src/entities/User.ts

```diff
@@ -10,9 +10,38 @@ export class User extends Node<User> {
   @Property({ columnType: 'text' })
   name: string;
 
-  constructor({ name, ...extraValues }: UserConstructorValues) {
+  @Property({ columnType: 'text' })
+  pronouns: string;
```

Comment:
@SpencerKaiser or @johnkahn should this be a string array or just be kept as a string? Dre and I could see arguments both ways so we wanted to ask y'all first

Reply:
Just a string, it'll be a freeform text input

Reply:
I'd say it should be optional though.

```suggestion
  pronouns?: string;
```